web/WebDevelopersGuide.md

```diff
@@ -599,7 +605,32 @@ return a new Sort instance.
 ###### Properties
  * `Column column` - The column which is sorted.
  * `String direction` - The direction of this sort, either `ASC`, `DESC`, or `REVERSE`.
- * `boolean isAbs` - True if the absolute value of the column should be used when sorting; defaults to false.
+ * `boolean isAbs` - True if the absolute value of the column should be used when sorting; defaults to false. 
+
+##### Abstract Class `CustomColumn`
+ * `String method` - The type of select method to use, either `VIEW` or `SELECT`. Defaults to `VIEW`.
```

Comment:
this only applies to formulas, any format/style should always use view i _think_

Reply:
OTOH, all subtypes have name, so perhaps name belongs in this type?

Reply:
Yea, the only hesitation with putting `name` in this type is that it's a little different in each context... either referencing an existing column (eg. `CustomColumnColor`), or creating a new column (`CustomColumnFormula`). Does that matter?

Reply:
I've made it clear from the docs what the `name` field means in each case. I've also added `Table.formatRow` and `Table.customColumn` methods to create `CustomColumnColor` (with `name=*`) and `CustomColumnFormat` respectively